### PR TITLE
Spin up a very basic version of multipage flow for 2020

### PR DESCRIPTION
**Pull request by kicferk1**
### Context

More of an exploratory experience - figuring out how far we can push the multistep form, what we can save, what screens make sense, etc. I would not merge it, the PR is mostly for the review app. 

### How can I view this in a review app?
Sign in as `school-leader@example.com`.
Go to `/schools/999999-example-school/year-2020/add` in the url.

Known things to address:
1. 'Opt out' somewhere, perhaps the first page.
2. Multiple participants.
3. Multiple schools.
3. Saving data correctly.
4. Hide behind a feature flag?
4. Tests.

### CODE REVIEW

File: app/views/schools/year2020/confirm.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <dd class="govuk-summary-list__actions">
+          <%# govuk_link_to issue: https://github.com/DFE-Digital/govuk-components/issues/193 %>
+          <%= govuk_link_to({ action: :show, step: "details" }, button: false) do %>
+            Change <span class="govuk-visually-hidden">personal details</span>
+          <% end %>
+        </dd>
+      </div>
+    </dl>
+
+<!--    QQQQ Add another participant, display more participants -->
```
Comment (timabell) [inline app/views/schools/year2020/confirm.html.erb:27]: what does `QQQQ` mean?

File: spec/forms/schools/add_participant_form_spec.rb [hunk 2]
```diff
@@ -57,7 +57,7 @@
     end
 
     context "when the email is not already in use" do
-      it "returns true" do
+      it "returns false" do
```
Comment (timabell) [inline spec/forms/schools/add_participant_form_spec.rb:60]: this doesn't make much more sense to me (in relation to the body of the test), and I don't understand why the logic has reversed

Comment (kicferk1) [reply spec/forms/schools/add_participant_form_spec.rb:60]:
```
describe "email_already_taken?" do
  context "when the email is not already in use" do
    it "returns false"
      ...
    end
  end
end
```
makes a lot of sense to me. The method checking if an email is already taken should return false when the email is not already in use. 

The description was simply incorrect - the logic hasn't changed (and the test contents haven't either).

File: app/views/schools/year2020/confirm.html.erb (hunk 1 above)
Comment (kicferk1) [reply app/views/schools/year2020/confirm.html.erb:27]: It means "resolve it before you merge", I use it to make sure I can quickly find those in my code :P

**Comment by kicferk1** [comment]
Closing in favor of https://github.com/DFE-Digital/early-careers-framework/pull/743

**Pull request closed without merging**